src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -3,25 +3,114 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 
-[assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(CopyColumnsTransform), typeof(CopyColumnsTransform.Arguments), typeof(SignatureDataTransform),
-    CopyColumnsTransform.UserName, "CopyColumns", "CopyColumnsTransform", CopyColumnsTransform.ShortName, DocName = "transform/CopyColumnsTransform.md")]
+[assembly: LoadableClass(CopyColumnsTransformer.Summary, typeof(IDataTransform), typeof(CopyColumnsTransformer),
+    typeof(CopyColumnsTransformer.Arguments), typeof(SignatureDataTransform),
+    CopyColumnsTransformer.UserName, "CopyColumns", "CopyColumnsTransform", CopyColumnsTransformer.ShortName,
+    DocName = "transform/CopyColumnsTransformer.md")]
 
-[assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(CopyColumnsTransform), null, typeof(SignatureLoadDataTransform),
-    CopyColumnsTransform.UserName, CopyColumnsTransform.LoaderSignature)]
+[assembly: LoadableClass(CopyColumnsTransformer.Summary, typeof(IDataView), typeof(CopyColumnsTransformer), null, typeof(SignatureLoadDataTransform),
+    CopyColumnsTransformer.UserName, CopyColumnsTransformer.LoaderSignature)]
+
+[assembly: LoadableClass(CopyColumnsTransformer.Summary, typeof(CopyColumnsTransformer), null, typeof(SignatureLoadModel),
+    CopyColumnsTransformer.UserName, CopyColumnsTransformer.LoaderSignature)]
+
+[assembly: LoadableClass(CopyColumnsTransformer.Summary, typeof(CopyColumnsRowMapper), null, typeof(SignatureLoadRowMapper),
+    CopyColumnsTransformer.UserName, CopyColumnsRowMapper.LoaderSignature)]
 
 namespace Microsoft.ML.Runtime.Data
 {
-    public sealed class CopyColumnsTransform : OneToOneTransformBase
+    public sealed class CopyColumnsEstimator : IEstimator<CopyColumnsTransformer>
+    {
+        private readonly (string Source, string Name)[] _columns;
+        private readonly IHost _host;
+
+        public CopyColumnsEstimator(IHostEnvironment env, string input, string output) :
+            this(env, new (string, string)[1] { (input, output) })
```

Comment:
>new (string, string)[1] { (input, output) } [](start = 22, length = 43)

It's a `params`, right? So could we do `this(env, (input, output))` instead of this long thing?